lib/cache.js

```diff
@@ -35,7 +35,6 @@ module.exports.init = function (callback) {
     cacheEnabled = true;
     cache = new lru({
       max: config.questionRenderCacheMaxItems,
-      length: (n) => n.length,
```

Comment:
The above `max` option now controls the number of items, so a `length` isn't used. I think this was always the intent based on the name of the `questionRenderCacheMaxItems` config option.